docs-kits/kits/Industry Core Kit/Software Development View/page_aspect-models.mdx

```diff
@@ -293,13 +285,13 @@ Aspect model in GitHub:
       "key": "batchId"
     },
     {
-      "value": "123-0.740-3434-A",
-      "key": "manufacturerPartId"
+      "key": "manufacturerId",
```

Comment:
Looking at the batch aspect itself, I still see differences between the semantic model example and how we describe it here. What you did is an improvement, for sure: Would you also remove the resulting differences?
I guess this also applies to the other aspect models

Reply:
@eckardg regarding the "differences", do you mean changing the examples in the Aspect models?